app/javascript/packages/document-capture/components/document-capture-warning.tsx

```diff
@@ -22,20 +22,37 @@ interface DocumentCaptureWarningProps {
   hasDismissed: boolean;
 }
 
+function getIdType(unknownFieldErrors) {
+  const idType = unknownFieldErrors.find((error) => error.field === 'unexpected_id_type');
+  return idType ? idType.error : null;
+}
+
 type GetHeadingArguments = {
   isResultCodeInvalid: boolean;
   isFailedDocType: boolean;
   isFailedSelfie: boolean;
   isFailedSelfieLivenessOrQuality: boolean;
+  unexpectedIdTypeError: boolean;
+  unknownFieldErrors: FormStepError<{ front: string; back: string }>[];
   t: typeof I18n.prototype.t;
 };
 function getHeading({
   isResultCodeInvalid,
   isFailedDocType,
   isFailedSelfie,
   isFailedSelfieLivenessOrQuality,
+  unexpectedIdTypeError,
+  unknownFieldErrors,
   t,
 }: GetHeadingArguments) {
+  if (unexpectedIdTypeError) {
+    const idType = getIdType(unknownFieldErrors);
+    const isPassport = idType?.message === 'passport';
+    const heading = isPassport
```

Comment:
the logic here probably is confusing. if it is passport that means it is the unexpected_id_type so we show the driver's license text. maybe naming the variable wrongIdIsPassport or something would help?